### PR TITLE
 Fix -1777 Accepting Only Images for updating profile picture

### LINKED ISSUE #1777: Accepting Only Images for updating profile picture
**I'm submitting a ...**
- [x] bug report
- [x] feature request

Will work on it ??
Yes

### PR DESCRIPTION
**Pull request by sk9331657**
Fixes #1777

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Accepting only images

### CODE REVIEW

**Comment by sk9331657** [comment]
@yashLadha  Done. :smile:

**Comment by sk9331657** [comment]
@yashLadha  @ParthS007  Please review.

File: frontend/app/templates/create-badges.hbs [hunk 1]
```diff
@@ -92,6 +92,7 @@
                         </div>
 
                     </div>
+
```
Comment (gabru-md) [inline frontend/app/templates/create-badges.hbs]: please do no add empty lines here.
please make the changes so that we can merge this.
cheers! :+1:

**Comment by sk9331657** [comment]
@gabru-md  @yashLadha  Please merge this.

**Pull request merged**